TU20Bot/Handler.cs

```diff
@@ -155,50 +157,74 @@ public class Handler {
                     messageLink = link(message),
                     messageContent = message.Content,
                     
-                    promptId = prompt.Id
+                    promptId = prompt.Id,
+                    state = EventState.Draft
                 });
                 
                 // We also want to create an index for the collection so we can do text searching later.
                 await eventCollection.Indexes.CreateOneAsync(new CreateIndexModel<EventModel>(
                     Builders<EventModel>.IndexKeys.Text(x => x.messageContent)
                 ));
 
-                // Add the example reactions, including tag emojis and ✅.
+                /// Add the Confirm and Cancel Buttons first
+                // Add Check-button for confirming the request
+                await prompt.AddReactionAsync(new Emoji("✅"));
+
+                // Add X-button for canceling the request
+                await prompt.AddReactionAsync(new Emoji("❌"));
```

Comment:
[`AddReactionsAsync`](https://docs.stillu.cc/api/Discord.MessageExtensions.html#Discord_MessageExtensions_AddReactionsAsync_Discord_IUserMessage_Discord_IEmote___Discord_RequestOptions_) is a handy helper method you can use here :)